pkg/network/tracer/tracer.go

```diff
@@ -315,11 +338,55 @@ func (t *Tracer) storeClosedConnections(connections []network.ConnectionStats) {
 	t.state.StoreClosedConnections(connections)
 }
 
+func monotonicToWallTime(t uint64) (uint64, error) {
```

Comment:
Can you look at https://github.com/DataDog/datadog-agent/pull/12475 and make sure this function is still correct.

Reply:
I copied over `TimeResolver` from `pkg/security/probe` for now, but I'm going to move that to `pkg/util` in a separate PR.